pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -335,7 +335,7 @@ func (o *Options) Run() error {
 
 		if cv.Spec.Channel != "" {
 			if cv.Spec.Upstream == "" {
-				fmt.Fprint(o.Out, "Upstream is unset, so the cluster will use an appropriate default.\n")
+				fmt.Fprint(o.Out, "Upstream is unset, so the cluster will use the default.\n")
```

Comment:
I had a debate about this with @LalatenduMohanty internally as a result of [this thread][1].  I should have copied a summary back up into the public comment then, but the gist is that [the API docs][2] have:

> By default it will use the appropriate update server for the cluster and region.

So I'd prefer to follow that example and remain ambiguous about the number of default values the CVO might be choosing from.

[1]: https://github.com/openshift/oc/pull/576#discussion_r647561401
[2]: https://github.com/openshift/api/blob/a6156965faae5ce117e3cd3735981a3fc0e27e27/config/v1/types_cluster_version.go#L59-L60

Reply:
Reverted to use "an appropriate default"